internal/appsec/ruleset_builder.go

```diff
@@ -166,6 +166,7 @@ func (r *rulesManager) compile() {
 		r.latest.Overrides = append(r.latest.Overrides, v.Overrides...)
 		r.latest.Exclusions = append(r.latest.Exclusions, v.Exclusions...)
 		r.latest.Actions = append(r.latest.Actions, v.Actions...)
+		r.latest.RulesData = append(r.latest.RulesData, v.RulesData...)
```

Comment:
so we were losing the rules data, right? 😅
if this is so, it clearly shows important missing the system tests.

Reply:
ca0396063c5150